docs/02.API-REFERENCE.md

```diff
@@ -169,6 +169,28 @@ typedef struct
 } jerry_property_descriptor_t;
 ```
 
+## jerry_heap_stats_t
+
+**summary**
+
+Description of JerryScript heap memory stats.
+It is for memory profiling.
+
+**Prototype**
+
+```c
+typedef struct
+{
+  size_t version /**< the version of the stats struct */
+  size_t size; /**< heap total size */
+  size_t allocated_bytes; /**< currently allocated bytes */
+  size_t peak_allocated_bytes; /**< peak allocated bytes */
+  size_t waste_bytes; /**< bytes waste due to blocks filled partially */
+  size_t peak_waste_bytes; /**< peak wasted bytes */
```

Comment:
Ok, I will remove it in this patch. If someday we really need it, let's put it into another PR.